src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -169,6 +173,7 @@ export default function ConfirmSwapModal({
   fiatValueInput: { data?: number; isLoading: boolean }
   fiatValueOutput: { data?: number; isLoading: boolean }
 }) {
+  const { chainId } = useWeb3React()
```

Comment:
is it accurate to use the wallet's chainId here? doesnt the swap flow have its own internal chainId in case there's a difference between the two?

Reply:
yes, but the user shouldn't be able to reach this part of the flow with these 2 values being unsynced. i think i can switch to use the contextual chainId without an change in behavior though.